scripts/ci/check_compliance.py

```diff
@@ -723,6 +723,30 @@ def parse_kconfig(self):
             # Clean up the temporary directory
             shutil.rmtree(kconfiglib_dir)
 
+    def module_kconfigs(self, regex):
+        manifest = Manifest.from_file()
+        kconfigs = ""
+
+        for project in manifest.get_projects([]):
+            if not manifest.is_active(project):
+                continue
+
+            if not project.is_cloned():
+                continue
+
+            module_path = PurePath(project.abspath)
+            module_samples = module_path.joinpath('samples')
```

Comment:
Tests and samples are properties of the module yaml files. Shouldn't we take those into account?

Reply:
They are? This is news to me... I will have a look